docs/src/main/paradox/running.md

```diff
@@ -0,0 +1,63 @@
+# Running a Projection
+
+Once you have decided how you want to build your projection, the next step is to run it. Typically, you run it in a distributed fashion in order to spread the load over the different nodes in an Akka Cluster. However, it's also possible to run it as a single instance (when not clustered) or as single instance in a Cluster Singleton.
+
+## Dependencies
+
+To distribute the projection over the cluster we recommend the use of [SharderDaemonProcess](https://doc.akka.io/docs/akka/current/typed/cluster-sharded-daemon-process.html). Add the following dependency in your project if not yet using Akka Cluster Sharding:
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-cluster-sharding-typed_$scala.binary.version$
+  version=2.6.5
+}
+
+Akka Projections require Akka 2.6.5 or later, see Akka version.
+
+For more information on using Akka Cluster consult Akka's reference on [Akka Cluster](https://doc.akka.io/docs/akka/current/typed/index-cluster.html) and [Akka Cluster Sharding](https://doc.akka.io/docs/akka/current/typed/cluster-sharding.html).
+
+## Sharded Daemon Process Example
+
+The Sharded Daemon Process can be used to distribute the `n` instances of a given Projection across the cluster. Therefore, it's important that each Projection instance consumes a subset of the stream of envelopes.
+
+How the subset is created depends on the kind of source we consume. If it's a Alpakka Kafka source, this is typically done using Kafka partitions. When consuming from Akka Persistence Journal, the events must be tagged with a shard number as demonstrate in the example bellow.
```

Comment:
"shard number", pretty confusing to use shard here. "slice" ;-)

Reply:
Yeah, I know. I knew we were going to discuss this. I did chose for shard because at the end of the day we use that term when talking about Cluster in Akka. That's everywhere. 

And in this example we will show how to use to configure the `ShardedDaemonProcess`. Even the code fragment that decides we select the tag is using a shard function like we show in other parts of akka dosc.

I did use the word slice on another place. ;-)

Anyway, I will change. But I think we should think more if its worth introducing another name next to shard.